go.mod

```diff
@@ -87,7 +87,7 @@ require (
 	github.com/juju/version/v2 v2.0.1
 	github.com/juju/viddy v0.0.0-beta5
 	github.com/juju/webbrowser v1.0.0
-	github.com/juju/worker/v3 v3.2.0
+	github.com/juju/worker/v3 v3.2.1-0.20230626153735-e43ac123ef3c
```

Comment:
Shouldn't we have a proper release of this?

Reply:
I've just circled back around and pinned it to `v3.3.0`